x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/routes.tsx

```diff
@@ -51,6 +47,13 @@ CloudSecurityPosture.displayName = 'CloudSecurityPosture';
 export const routes: SecuritySubPluginRoutes = [
   {
     path: CLOUD_SECURITY_POSTURE_BASE_PATH,
-    component: CloudSecurityPosture,
+    component: withSecurityRoutePageWrapper(
+      CloudSecurityPosture,
+      SecurityPageName.cloudSecurityPostureDashboard,
```

Comment:
@ashokaditya I think it should be `cloud_security_posture` not `cloud_security_posture_dashboard` 👍

Reply:
Ok, your call 👍 Thanks!

Reply:
@tomsonpl What page and URL do you see when you navigate to `/app/security/cloud_security_posture/dashboard` on a complete/essentials security serverless project?

Reply:
Oh no, I think it does the same thing. The question is if that has a purpose? I prefer to make as little changes outside of the scope as possible. But you do your thing, all good 👍

Reply:
Yeah it kind of does the same thing with your change as well. I am guessing that's why there isn't a `cloud_security_posture` page name already. Nevertheless the tests pass when I revert the default route to be `cloud_security_posture-dashboard` along with `omitSpyRoute`. I believe we will learn more when we get this reviewed.